apps/dashboard/src/components/editor/index.js

```diff
@@ -55,11 +58,9 @@ const Editor = ( { projectId } ) => {
 		debounce( ( content ) => {
 			try {
 				const blocks = parse( content );
-				const currentProject = project || {};
+
 				saveAndUpdateProject( projectId, {
-					...currentProject,
```

Comment:
The `saveAndUpdateProject` method will create or update a project based on its existence. Since it will patch the data, do we need to send data other than what is actually being modified?
This will also help to prevent sending outdated data.